wp-includes/class-wp-web-app-manifest.php

```diff
@@ -677,4 +707,37 @@ function updateShortNameField() {
 		</script>
 		<?php
 	}
+
+	/**
+	 * Load 'wp/site-logo'-block filter
+	 *
+	 * Handle the 'site_icon_maskable' setting in a full-site-editing context,
+	 * by enqueing a filter to the 'wp:site-logo' block within the site-editor.
+	 *
+	 * @since  0.8.0-alpha
+	 * @see    https://developer.wordpress.org/reference/hooks/enqueue_block_editor_assets/
+	 *
+	 * @return void
+	 */
+	public function enqueue_site_icon_maskable_block_editor_assets() {
+		$dir  = '/wp-includes/js/dist';
+		$path = PWA_PLUGIN_DIR . $dir;
+
+		// Stop, if needed file was not built successfully.
+		$script_asset_path = "$path/site-icon-maskable.asset.php";
+		if ( ! file_exists( $script_asset_path ) ) {
+			return;
+		}
+
+		$script_asset = require $script_asset_path;
+		$index_js     = "$dir/site-icon-maskable.js";
+		wp_enqueue_script(
+			'pwa-site-icon-maskable-block-editor',
+			plugins_url( $index_js, __FILE__ ),
```

Comment:
I'm getting a 404 for the script, with the URL including `...wp-includes/wp-includes...`. This is due to the plugin file not being used here as expected:

```suggestion
			plugins_url( $index_js, PWA_PLUGIN_FILE ),
```

Reply:
Yes, sorry.